pyramid_openapi3/__init__.py

```diff
@@ -86,9 +80,7 @@ def openapi_validated(request: Request) -> dict:
 
     if request.environ.get("pyramid_openapi3.validate_request"):
         openapi_request = PyramidOpenAPIRequest(request)
-        validated = settings["request_validator"].validate(
-            settings["spec"], openapi_request
-        )
+        validated = settings["request_validator"].unmarshal(openapi_request)
```

Comment:
The `spec` is now provided when `settings["request_validator"]` is instantiated during Pyramid bootup.